### PR TITLE
Add info on breakpoints for CSS and Sass

### PR DESCRIPTION
**Pull request by ericwbailey**
This PR adds information about how thoughtbot approaches working with CSS breakpoints. It is sourced from [this conversation in Slack](https://thoughtbot.slack.com/archives/C02C7SZUQ/p1567010261001200).

### CODE REVIEW

File: best-practices/README.md [hunk 1]
```diff
@@ -232,8 +232,14 @@ CSS
   project-specific browser support that is needed.
 * Prefer `overflow: auto` to `overflow: scroll`, because `scroll` will always
   display scrollbars outside of macOS, even when content fits in the container.
+* [Create breakpoints](breakpoints) when the content "breaks," and is awkward or 
+  difficult to read,
+  * Avoid creating breakpoints that target specific devices
+  * Prefer `em` units instead of `px` for breakpoint values
+  * Start with a mobile breakpoint and work outwards using `min-width`/`min-height`
```
Comment (enatario) [inline best-practices/README.md]: Perhaps instead of specifying `mobile` here, we should remain agnostic and just say "smallest viewport" size?

Comment (ericwbailey) [reply best-practices/README.md]: Love it. I'll update that now.

**Comment by ericwbailey** [comment]
Thanks for the feedback!

**Comment by tysongach** [comment]
Just a friendly reminder, [from the readme](https://github.com/thoughtbot/guides#contributing):

> if you have commit access, please don't merge changes without waiting a week for everybody to leave feedback.

**Comment by ericwbailey** [comment]
Oh crap. Sorry about that.

**Pull request merged**